### PR TITLE
Build on + revise @jvonau's PR #1994 ("Combine iiab-support & install-support") for OpenVPN

### PR DESCRIPTION
**Pull request by holta**
2 primary changes, above & beyond PR #1994:

- I prefer `/usr/bin/iiab-support` in place (not just a symlink) so that it generally works even when `/opt/iiab/iiab` disappears for any reason e.g. if the implementer chose to conserve disk/storage space, or for any other reason.

- Central file `roles/openvpn/tasks/main.yml` and directory `roles/openvpn/templates/` have been extensively pruned of older/deprecated/unused "bitrot"

### CODE REVIEW

File: iiab-support [hunk 1]
```diff
@@ -46,14 +49,17 @@ fi
...
-    echo -e "Please wait a few minutes as IIAB Stage 1 (1-prep) & OpenVPN are installed...\n"
+    echo -e "Plz wait a few minutes as sshd, iiab-admin & OpenVPN are confirmed/installed...\n"
     if grep -q '^openvpn_install:' /etc/iiab/local_vars.yml; then
         sed -i "s/^openvpn_install:.*/openvpn_install: True/" /etc/iiab/local_vars.yml
     else
         echo "openvpn_install: True" >> /etc/iiab/local_vars.yml
     fi
     cd /opt/iiab/iiab
-    ./runrole 1-prep
+    export ANSIBLE_LOG_PATH="/opt/iiab/iiab/iiab-install.log"
```
Comment (jvonau) [inline iiab-support]: if you use the full path the cd is not required

File: roles/openvpn/tasks/main.yml [hunk 2]
```diff
@@ -82,19 +80,16 @@
...
-    # Obsolete & unused for ~2 years as of August 2018:
-    #- { src: 'iiab-vpn.j2', dest: '/usr/bin/iiab-vpn', mode: '0755' }
+    # 2019-10-09: Not recommended for over a year & no longer in use
+    # - { src: 'iiab-handle.j2', dest: '/usr/bin/iiab-handle', mode: '0755' }
+
+- name: Copy /opt/iiab/iiab/iiab-support to /usr/bin/iiab-support, in case git tree deleted e.g. on a smaller IIAB install
+  copy:
+    src: "{{ iiab_dir }}/iiab-support"
+    dest: /usr/bin/
+    mode: 0755
```
Comment (jvonau) [inline roles/openvpn/tasks/main.yml:94]: if the git tree is deleted, then none of the ansible routines are available

Comment (holta) [reply roles/openvpn/tasks/main.yml:94]: True.  If there are any especially important scripts that need to elevated from `/opt/iiab/iiab/scripts` or `/opt/iiab/iiab` to `/usr/bin` LMK and I will do that.

But we might possibly be ok as is?

File: iiab-support (hunk 1 above)
Comment (holta) [reply iiab-support]: Good to know.  I've commented it out.

File: roles/openvpn/tasks/main.yml (hunk 2 above)
Comment (jvonau) [reply roles/openvpn/tasks/main.yml:94]: Think I see what you want with /usr/bin/iiab-support post install but the log directory must exist, a gratuitous "mkdir -p" in place of "cd" can't hurt.

File: iiab-support (hunk 1 above)
Comment (jvonau) [reply iiab-support]: Here

File: roles/openvpn/tasks/main.yml (hunk 2 above)
Comment (jvonau) [reply roles/openvpn/tasks/main.yml:94]: I have to backtrack, the cd /opt/iiab/iiab is required to find $INVENTORY $PLAYBOOK

Comment (jvonau) [reply roles/openvpn/tasks/main.yml:94]: I have to backtrack, the cd /opt/iiab/iiab is required to find $INVENTORY $PLAYBOOK

Comment (jvonau) [reply roles/openvpn/tasks/main.yml:94]: I have to backtrack, the cd /opt/iiab/iiab is required to find $INVENTORY $PLAYBOOK

Comment (holta) [reply roles/openvpn/tasks/main.yml:94]: Ok.  See https://github.com/holta/iiab/blob/support_yml/iiab-support#L59-L70

All code paths tested.

Should be ready to merge.

**Pull request merged**